Implements Acopio List screen

manue1 opened the pull request:
- Adds a new `/list` route displaying the List screen
- introduces `displayProducts` prop that's passed down to AcopioCard to decide if products are shown; if not the direction of the Acopio is shown with a link to it on Google Maps

This fixes #64

jeduan commented on src/screens/List.js:
```diff
@@ -0,0 +1,76 @@
+        this.setState({
+          isLoading: true
+        })
+        return getAcopios()
```
por qué cargarlos dos veces?

manue1 replied:
Se me pasó :sleeping:

manue1 commented:
@jeduan I have removed the erroneous second API call and updated the AcopioCard to only show products if there are any available, if not an info message is shown.
